### PR TITLE
docs: passwordless sms login

### PR DESCRIPTION
**Pull request by aeneasr**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: docs/kratos/passwordless/06_code.mdx [hunk 1]
```diff
@@ -29,37 +30,42 @@ import BrowserWindow from "@site/src/theme/BrowserWindow"
 In the registration flow below we are only presented an input field for the email address, as we have not defined any other traits
```
Comment (piotrmsc) [inline docs/kratos/passwordless/06_code.mdx:30]: nit: below there is no registration view, only direct new configuration section you have added and above there is only a sign-in screen example so the old text does not match the current layout.
Screenshot 2024-10-08 at 08 52 18

**Review by piotrmsc — COMMENTED** [review]
nit: do w have any (our) documentation/blogpost that we can link as warning explaining why one should not use OTP via text message?

**Comment by aeneasr** [comment]
Thanks @piotrmsc - addressed your feedback

**Review by vinckr — APPROVED** [review]
lgtm

**Pull request merged**